domains/brandon.json

```diff
@@ -1,12 +1,11 @@
 {
   "description": "Brandons's personal developer website",
-  "repo": "https://github.com/b-hexsoul",
+  "repo": "https://github.com/b-hexsoul/b-hexsoul.github.io",
   "owner": {
     "username": "b-hexsoul",
     "email": "brandon.hexsel@gmail.com"
   },
   "record": {
-    "CNAME": "b-hexsoul.github.io",
-    "URL": "https://brandon.is-a.dev"
+    "CNAME": "https://b-hexsoul.github.io/portfolio/",
```

Comment:
CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `b-hexsoul.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).